### PR TITLE
initial update of detox to version 20 + comments on failing tests

### PR DESCRIPTION
**Pull request by mwieja0**
E2E tests are now using latest Detox and Jest.

Tested only for iOS simulator at this moment.

Failing tests are skipped, but I carried out initial investigation and left FIXME comments with context.

### CODE REVIEW

File: e2e/registerENSFlow.spec.js [hunk 1]
```diff
@@ -281,8 +281,11 @@ describe('Register ENS Flow', () => {
     await Helpers.tapByText('CryptoKitties');
     await Helpers.tapByText('Arun Cattybinky');
     await Helpers.checkIfVisible('ens-assign-records-review-action-button');
-    await Helpers.delay(2000);
+    console.log('found the button');
```
Comment (jinchung) [inline e2e/registerENSFlow.spec.js:284]: cleanup consoles

**Comment by jinchung** [comment]
TODO:
1. update mac stadium machines detox and detox-cli versions
2. update instructions for devs to ensure the tests can run locally for them: removing and installing latest detox-cli
3. removing and reinstalling pods
4. detox cache cleaning

**Comment by jinchung** [comment]
don't forget to add yarn.lock alongside the package.json changes

File: src/utils/__tests__/ens.test.ts [hunk 2]
```diff
@@ -3,31 +3,31 @@ import { validateENS } from '../ens';
 describe('valid names', () => {
   it('domain', () => {
     expect(validateENS('lol.eth')).toMatchInlineSnapshot(`
-      Object {
+      {
```
Comment (jinchung) [inline src/utils/__tests__/ens.test.ts:6]: removing the 'Object' as the updated jest version supports a new format for inline snapshots

File: e2e/homeScreen.spec.js [hunk 3]
```diff
@@ -47,7 +47,7 @@ describe('Home Screen', () => {
     await Helpers.swipe('wallet-screen', 'up', 'slow', 0.4);
     await Helpers.checkIfVisible('profile-sticky-header');
     await Helpers.checkIfExistsByText(RAINBOW_TEST_WALLET);
-    await Helpers.swipe('wallet-screen', 'down');
+    await Helpers.swipe('wallet-screen', 'down', 'slow', 0.4);
```
Comment (jinchung) [inline e2e/homeScreen.spec.js:50]: we were observing the "swipe down" not successfully being able to swipe down unless we added the slow param

File: e2e/jest.e2e.config.js [hunk 4]
```diff
@@ -6,9 +6,13 @@ module.exports = {
   maxWorkers: 1,
   setupFilesAfterEnv: ['./init.js'],
   testEnvironment: './environment',
+
```
Comment (jinchung) [inline e2e/jest.e2e.config.js:9]: changes below are due to detox 20 version update

**Review by skylarbarrera — APPROVED** [review]
this is awesome ty so much 🙏

**Pull request merged**